Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/GameModeBase.h"
+#include "UserExperienceReporter.h"
+
+#include "BenchmarkGymGameModeBase.generated.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogBenchmarkGymGameModeBase, Log, All);
+
+UCLASS()
+class GDKTESTGYMS_API ABenchmarkGymGameModeBase : public AGameModeBase
+{
+	GENERATED_BODY()
+public:
+	ABenchmarkGymGameModeBase();
+
+	UFUNCTION(BlueprintCallable)
```

Comment:
Why are these exposed?

Reply:
We had functionality in the Offloading gym game mode to dynamically update when the worker flag value changes. I wanted to ensure we use the same keys. I might take a look at the logic inside offloading game mode and see if it can be simplified.